_posts/2020-08-11-backports-application.adoc

```diff
@@ -0,0 +1,113 @@
+---
+layout: post
+title: From zero to hero on backporting pull-requests
+date: 2020-08-11
+tags: user-story
+author: gastaldi
+---
+= From zero to hero on backporting pull-requests
+
+Before we dig into the solution, let me give you a quick explanation about our release process
+
+== Release process
+=== Minor versions
+Our master branch is always ready for the next minor release. That means that any pull-requests merged in there are available in the next minor release. This process is usually very smooth and involves no backporting.
+
+=== Patch releases or CRs
+Whenever a new patched version (or a CR) is about to be released in Quarkus, our Release engineer starts backporting commits from pull-requests that were merged in the master branch.
```

Comment:
```suggestion
Whenever a new patched version (or a CR) is about to be released in Quarkus, our Release engineer starts backporting commits from pull-requests that were merged in the master branch.
```

The first CR is fine. That's all the subsequent releases that needs backporting (`.0.Final` included)